app/src/main/java/camp/nextstep/edu/github/view/RecyclerViewAdapter.kt

```diff
@@ -0,0 +1,19 @@
+package camp.nextstep.edu.github.view
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import camp.nextstep.edu.github.databinding.ItemRepoBinding
+import camp.nextstep.edu.github.domain.RepositoryVO
+
+class RecyclerViewAdapter : ListAdapter<RepositoryVO, RecyclerViewHolder>(ViewDiffCallback()) {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerViewHolder {
+        val inflate = LayoutInflater.from(parent.context)
+        val binding: ItemRepoBinding = ItemRepoBinding.inflate(inflate, parent, false)
```

Comment:
```suggestion
        val binding: ItemRepoBinding = ItemRepoBinding.inflate(LayoutInflater.from(parent.context), parent, false)
```
inflate라는 변수는 사용하는 곳이 한 곳밖에 없네요.
그렇다면 해당 변수의 값을 인라인해보면 어떨까요?